hasher-matcher-actioner/hmalib/lambdas/api/datasets_api.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import bottle
+import typing as t
+from dataclasses import dataclass, asdict
+from .middleware import jsoninator, JSONifiable
+from hmalib.common.config import HMAConfig
+from hmalib.common import config as hmaconfig
+from hmalib.lambdas.fetcher import *
+
+
+@dataclass
+class Dataset(JSONifiable):
```

Comment:
Whats the difference between this class and ThreatExchangeConfig. Can we reuse that?

Reply:
this is class is created to easily converted to or from json, looks we can't reuse ThreatExchangeConfig